rpcs3/rpcs3qt/trophy_manager_dialog.cpp

```diff
@@ -321,7 +321,16 @@ trophy_manager_dialog::trophy_manager_dialog(std::shared_ptr<gui_settings> gui_s
 		m_game_combo->setCurrentText(m_game_table->item(m_game_table->selectedItems().first()->row(), GameColumns::GameName)->text());
 	});
 
-	RepaintUI(true);
+	RepaintUI();
+
+	// Fill trophy list
+	auto loadProgress = new QProgressDialog(
```

Comment:
i'd like to see that as a separate function, so that it can be called by other user interactions in the future.
Also, closing the application during the loading phase will result in an error. So please abort this thread before the trophy manager is destroyed.